diff-ex-and-viz.md

```diff
@@ -308,18 +401,37 @@ pheatmap(assay(vsd)[genes, ], cluster_rows=TRUE, show_rownames=TRUE,
          cluster_cols=FALSE, annotation_col=annot_col)
 ```
 
+<center><img src="_static/deseq_heatmap.png" width="90%"></center>
+<br>
+
 We see that our samples do cluster by condition, but that by looking at just the
 counts, the patterns aren't very strong. How does this compare to our MDS plot?
 
 **Question**  
 When are heatmaps useful?   
 What other types of heatmaps have you seen in the wild?
 
+## Judging DGE results
+
+Once you have obtained a table of genes that show signs of differential expression, you have reached one of the most important milestones of RNA-seq analysis!. To evaluate how confident you can be in that list of DE genes, you should look at several aspects of the analyses you did and perform basic checks on your results:
+
+1. Did the unsupervised clustering and PCA analyses reproduce the major trends of the initial experiment? For example, did replicates of the same condition cluster together and were they well separated from the replicates of the other condition(s)?
+
+2. How well do different DGE programs agree on the final list of DE genes? You may want to considerperforming downstream analyses only on the list of genes that were identified as DE by more than one tool.
```

Comment:
I'm not a fan of questions without answers. If the instructor doesn't know where the mindset of the question creator was, it can leave people making things up on the fly. And if people are learning from it on their own, they are left not knowing if what they are thinking is correct or not.